### PR TITLE
Changed Ascii-Logo

### PR DESCRIPTION
**Pull request by kcamcam**
because famingo labs deserves a new look
used http://www.network-science.de/ascii/ to make it

### CODE REVIEW

**Comment by kcamcam** [comment]
damn true

File: index.html [hunk 1]
```diff
@@ -16,16 +16,27 @@
...
+88                                                                            
+88aaaaa ,adPPYYba, 88,dPYba,,adPYba,  88 8b,dPPYba,   ,adPPYb,d8  ,adPPYba,   
+88""""" ""     `Y8 88P'   "88"    "8a 88 88P'   `"8a a8"    `Y88 a8"     "8a  
+88      ,adPPPPP88 88      88      88 88 88       88 8b       88 8b       d8  
+88      88,    ,88 88      88      88 88 88       88 "8a,   ,d88 "8a,   ,a8"  
+88      `"8bbdP"Y8 88      88      88 88 88       88  `"YbbdP"Y8  `"YbbdP"'   
+                                                      aa,    ,88              
+                                                       "Y8bbdP"                                                             
+88                     88                     
+88                     88
```
Comment (almiche) [inline index.html]: Agreed this is way to big dawg

**Comment by kcamcam** [comment]
damn the dream is over

**Comment by kcamcam** [comment]
famingo
how bou da

**Comment by almiche** [comment]
Where did the ascii go famingo

**Comment by almiche** [comment]
Looks hella lit tho

**Comment by almiche** [comment]
"Although I liked the old one, I trust your design skills" - Leger 2017

**Pull request closed without merging**